crates/sui-protocol-config/src/snapshots/sui_protocol_config__test__Mainnet_version_37.snap

```diff
@@ -207,7 +206,7 @@ check_zklogin_id_cost_base: 200
 check_zklogin_issuer_cost_base: 200
 scoring_decision_mad_divisor: 2.3
 scoring_decision_cutoff_value: 2.5
-execution_version: 3
+execution_version: 2
```

Comment:
how did the wrong execution version end up in the release branch?

Reply:
that is so bad... sorry...
I forced execution version to a previous protocol config just to try the fix and I do not understand how it got back into the PR. 
I thought I did that on my machine only, but obviously....